### ISSUE TITLE
Drop table mappings support

### ISSUE THREAD

**Issue by jimkont** [dbpedia/extraction-framework#345]
Should we drop support for Table mappings in the framework? This didn't work as expected so far

http://mappings.dbpedia.org/index.php?title=Special%3APrefixIndex&prefix=Table&namespace=204
http://mappings.dbpedia.org/index.php/How_to_edit_DBpedia_Mappings#How_to_map_a_Wikipedia_Table

**Comment by jcsahnwaldt** [comment]
Yeah, that was in idea for the project focusing on extracting better and more data for car-related stuff. I wouldn't mind if it was removed.

**Comment by VladimirAlexiev** [comment]
Please don't drop! We're getting into some Automotive stuff (brands, races, series etc) and can work on fixing whatever doesn't work.

TableMap | Wikipedia | Sample | Notes
----- | ----- | ------ | -------
[Automobile](http://mappings.dbpedia.org/index.php/Mapping_en:Table/Automobile) | [BMW_3_Series](https://en.wikipedia.org/wiki/BMW_3_Series) | [extract](http://mappings.dbpedia.org/server/extraction/en/extract?title=BMW_3_Series&revid=&format=turtle-triples&extractors=custom) |engines & sales. Fixed a bit
[Race](http://mappings.dbpedia.org/index.php/Mapping_en:Table/Race) | [2015_Firestone_Grand_Prix_of_St._Petersburg](https://en.wikipedia.org/wiki/2015_Firestone_Grand_Prix_of_St._Petersburg) | [extract](http://mappings.dbpedia.org/server/extraction/en/extract?title=2015_Firestone_Grand_Prix_of_St._Petersburg&revid=&format=turtle-triples&extractors=custom) |race results
[Championship](http://mappings.dbpedia.org/index.php/Mapping_en:Table/Championship) | [2015_IndyCar_Series](https://en.wikipedia.org/wiki/2015_IndyCar_Series) | [extract](http://mappings.dbpedia.org/server/extraction/en/extract?title=2015_IndyCar_Series&revid=&format=turtle-triples&extractors=custom) |championship teams, drivers

I fixed a bit the Automobile map and it produces results on the indicated sample (though attaches the Sales numbers to BMW 3 Series (F30/F31) which is the last infobox on the page, rather than to BMW 3 Series overall).

Unfortunately I have no luck with the other 2 maps I created. I suspect the reason is that the tables in the indicated sample pages have a more complex structure:
- [2015_Firestone_Grand_Prix_of_St._Petersburg#Race_results](https://en.wikipedia.org/w/index.php?title=2015_Firestone_Grand_Prix_of_St._Petersburg&action=edit&section=3) has some headers like `{{abbr|Pos|Position}}` instead of merely `Position`
- [2015_IndyCar_Series#Teams_and_drivers](https://en.wikipedia.org/wiki/2015_IndyCar_Series#Teams_and_drivers) has vertical cell spans

It's either this, or these two new table maps are somehow not enacted by the server?
@jimkont or @jcsahnwaldt please help!
